packages/manager/apps/dedicated/client/app/components/dedicated-cloud/users/iam/iam.controller.js

```diff
@@ -0,0 +1,78 @@
+export default class DedicatedCloudIamCtrl {
+  /* @ngInject */
+  constructor(
+    $q,
+    $translate,
+    DedicatedCloud,
+    ouiDatagridService,
+    ovhFeatureFlipping,
+    coreURLBuilder,
+  ) {
+    this.$q = $q;
+    this.$translate = $translate;
+    this.DedicatedCloud = DedicatedCloud;
+    this.ouiDatagridService = ouiDatagridService;
+    this.ovhFeatureFlipping = ovhFeatureFlipping;
+    this.coreURLBuilder = coreURLBuilder;
+  }
+
+  $onInit() {
+    this.loading = true;
+    return this.$q
+      .all({
+        featureAvailability: this.ovhFeatureFlipping
+          .checkFeatureAvailability('iam')
+          .then((featureAvailability) =>
+            featureAvailability.isFeatureAvailable('iam'),
+          ),
+        iamStatus: this.DedicatedCloud.getIamStatus(this.productId),
+      })
+      .then(({ iamStatus, featureAvailability }) => {
+        this.iamFeatureAvailability = featureAvailability;
+        this.iamEnabled = iamStatus?.state === 'enabled';
+        this.iamDisabled = iamStatus?.state === 'disabled';
+        this.iamStatus = iamStatus;
+
+        if (this.iamEnabled || this.iamDisabled) {
+          return this.DedicatedCloud.isIamTogglable(
+            this.productId,
+            this.iamStatus.state,
+            true,
+          )
+            .then((response) => {
+              this.iamTogglable = response.toggable;
+            })
```

Comment:
```suggestion
          return this.DedicatedCloud.isIamTogglable(
            this.productId,
            this.iamStatus.state,
          )
            .then((isIamTogglable) => {
              this.iamTogglable = isIamTogglable;
            })
```
https://github.com/ovh/manager/pull/9649#discussion_r1277304830